annotations/tests/applicators.json

```diff
@@ -0,0 +1,459 @@
+{
+  "$schema": "../test-suite.schema.json",
+  "description": "The applicator vocabulary",
+  "suite": [
+    {
+      "description": "`properties`, `patternProperties`, and `additionalProperties`",
+      "compatibility": "3",
+      "schema": {
+        "properties": {
+          "foo": {
+            "title": "Foo"
+          }
+        },
+        "patternProperties": {
+          "^a": {
+            "title": "Bar"
+          }
+        },
+        "additionalProperties": {
+          "title": "Baz"
+        }
+      },
+      "tests": [
+        {
+          "instance": {},
+          "assertions": [
+            {
+              "location": "/foo",
```

Comment:
What location does this represent? I assume the instance, but there's schema location. There's also evaluation path that we could consider.  I'm not sure evaluation path is necessary, but the other two should both be present, I think.

Secondly, it's inconsistent.  Some are uri-encoded, and others aren't.